### PR TITLE
Translation-ja: Add product-owner/ja/01-opening-article

### PR DESCRIPTION
**Pull request by ystk**
This is Japanese translation of the first article in Product Owner chapter which relate to #407 .

### CODE REVIEW

File: product-owner/ja/01-opening-article-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+== はじめに
+
+こんにちは、中間管理職やプロダクトオーナー、プログラムマネージャーの皆さん。
+もっと幸せで効果的なチームをリードしたいと思いますか？
+報告地獄に閉じ込められた記録係よりもリーダーになりたいですか？
+
```
Comment (johtani) [inline product-owner/ja/01-opening-article-ja.asciidoc:6]:
```suggestion
```
No need new line.

File: product-owner/ja/01-opening-article-ja.asciidoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+私の会社にInnerSourceのプロセスを導入したことで、約80%が割り込み駆動型開発だった複数の大規模システムのリファクタリングをすることができました。
+そして私たちは皆、その環境を管理すること、ましてや開発することが、いかに難しいかを知っています。
+
+InnerSourceプロセスを通じて、私達は数十年積み残していた内部残件を1年足らずで削減もしました。
```
Comment (johtani) [inline product-owner/ja/01-opening-article-ja.asciidoc:10]:
```suggestion
InnerSourceプロセスを通じて、私たちは数十年積み残していた内部残件を1年足らずで削減もしました。
```

File: product-owner/ja/01-opening-article-ja.asciidoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+InnerSouceの重要な要素はオープンであることです。
+これは、ほとんどの企業のチームが閉じ込められているサイロを破壊します。
```
Comment (johtani) [inline product-owner/ja/01-opening-article-ja.asciidoc:12]:
```suggestion
これは、ほとんどの企業のチームが陥っているサイロ化を破壊します。
```

**Review by johtani — CHANGES_REQUESTED** [review]
Great @ystk !
I left some comments and suggestions,
Please check it.

**Comment by ystk** [comment]
Thanks for your review, @johtani ! I applied all your suggestions.

**Review by johtani — APPROVED** [review]
LGTM! Thanks, @ystk

**Comment by rrrutledge** [comment]
Great work, all!

**Pull request merged**